docs/guide/persistence_cache.md

```diff
@@ -159,26 +159,26 @@ Out of the box in `config/packages/cache_pool/cache.redis.yaml` you'll find a de
 
 !!! note "Ibexa Cloud"
 
-    For Ibexa Cloud/Platform.sh: This is automatically configured in `bin/config/env/platformsh.php` if you have enabled Redis as `rediscache` Platform.sh service.
+    For Ibexa Cloud/Platform.sh: This is automatically configured in `vendor/platformsh/symfonyflex-bridge/platformsh-flex-env.php` if you have enabled Redis as `rediscache` Platform.sh service.
 
-For anything else, you can enable it with environment variables detected automatically by `bin/config/env/generic.php`.
+For anything else, you can enable it with environment variables.
 For instance, if you set the following environment variables `export CACHE_POOL="cache.redis" CACHE_DSN="secret@example.com:1234/13"`, it will result in config like this:
 
 ``` yaml
 services:
     cache.redis:
-        # NOTE: This optimized Redis Adapter is avaiable as of 2.5LTS via https://github.com/ezsystems/symfony-tools
```

Comment:
TODO: also fix "avaiable" typo in [2.5](https://github.com/ezsystems/developer-documentation/blob/2.5/docs/guide/persistence_cache.md?plain=1#L172).

Reply:
Done in b8859bfe9da1c15909784e7be5a409de76d4f619